assets/prefabs/items/CrudeShears.prefab

```diff
@@ -1,11 +1,16 @@
 {
   "parent": "engine:iconItem",
   "Item": {
-    "icon": "ManualLabor:ManualLabor#CrudeShears"
+    "icon": "ManualLabor:ManualLabor#CrudeShears",
+    "damageType": "shovelDamage"
```

Comment:
Why would shears have shovel damage?

Reply:
This question still stands

Reply:
Missed committing the prefab file, done now.